pyseir/rt/infer_rt.py

```diff
@@ -115,95 +106,79 @@ def _generate_input_data(
             "the Infection Rate Metric",
             region=regional_input.display_name,
         )
-        return pd.DataFrame()
+        return None
```

Comment:
What's the thought of making the return type optional instead of returning an empty timeseries?

Reply:
I like that since it's a series, it's a more atomic unit.  Feels like the resulting code paths make a bit more sense

Reply:
K. Seems like our personal opinions are different. I'm fine with it.